iOS/UI/Manga/MangaDetailHeaderView.swift

```diff
@@ -581,3 +586,11 @@ extension MangaDetailHeaderView: SizeChangeListenerDelegate {
         sizeChangeListener?.sizeChanged(bounds.size)
     }
 }
+
+extension UIButton {
```

Comment:
I don't think there's any reason to make this an extension on UIButtons, especially since it's only being used once. I would just move this code into the `configure()` function for the `safariButton`

Reply:
Thanks for the feedback. This piece of code was unnecessary, just changed it